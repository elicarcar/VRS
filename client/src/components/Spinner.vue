<template>
  <div class="text-center">
    <b-spinner label="Spinning"></b-spinner>
  </div>
</template>

<script>
export default {
  name: 'Spinner',
}
</script>
