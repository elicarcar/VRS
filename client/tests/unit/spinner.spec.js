import { shallowMount, createLocalVue } from '@vue/test-utils'
import BootstrapVue, { BootstrapVueIcons } from 'bootstrap-vue'

import Spinner from '@/components/Spinner.vue'

const localVue = createLocalVue()

localVue.use(BootstrapVue)
localVue.use(BootstrapVueIcons)

describe('Spinner', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Spinner)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
