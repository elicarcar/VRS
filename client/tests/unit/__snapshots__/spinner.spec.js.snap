// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<div class="text-center">
  <b-spinner label="Spinning"></b-spinner>
</div>
`;
